### PR TITLE
LQ Format setting in OSD

### PR DESCRIPTION
**Pull request by Kaiowarez**
* LQ format setting is OSD

### CODE REVIEW

**Comment by nerdCopter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=6303053&bid=20497711)

File: src/main/cms/cms_menu_osd.c [hunk 1]
```diff
@@ -44,25 +44,35 @@
...
 
+static const char * const cms_osdcrsfformat[] = {
+    "TBS", "MODE", "FREQ"
+};
+
 static long menuOsdActiveElemsOnExit(const OSD_Entry *self)
 {
     UNUSED(self);
 
+    memcpy(&osdConfigMutable()->lq_format, &osdConfig_one, sizeof(uint8_t));
```
Comment (nerdCopter) [inline src/main/cms/cms_menu_osd.c:63]: Codacy found an issue: [Does not check for buffer overflows when copying to destination (CWE-120).  Make sure destination can always hold the source data. ](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=6303053)

File: src/main/cms/cms_menu_osd.c [hunk 2]
```diff
@@ -44,25 +44,35 @@
 
 #ifdef USE_EXTENDED_CMS_MENUS
 static uint16_t osdConfig_item_pos[OSD_ITEM_COUNT];
-
+static uint8_t osdConfig_one;
 static long menuOsdActiveElemsOnEnter(void)
 {
+    memcpy(&osdConfig_one, &osdConfig()->lq_format, sizeof(uint8_t));
```
Comment (nerdCopter) [inline src/main/cms/cms_menu_osd.c:50]: Codacy found an issue: [Does not check for buffer overflows when copying to destination (CWE-120).  Make sure destination can always hold the source data. ](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=6303053)

**Comment by Kaiowarez** [comment]
Damn... I gotta double check it, I thought I got it working good.. But
thats not good...

**Comment by nerdCopter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=6303053&bid=20497711)

File: src/main/cms/cms_menu_osd.c [hunk 3]
```diff
@@ -44,25 +44,36 @@
...
 
+static const char * const cms_osdcrsfformat[] = {
+    "TBS", "MODE", "FREQ"
+};
+
 static long menuOsdActiveElemsOnExit(const OSD_Entry *self)
 {
     UNUSED(self);
 
+    memcpy(&osdConfigMutable()->lq_format, &osdConfig_one, sizeof(osdConfig_t));
```
Comment (nerdCopter) [inline src/main/cms/cms_menu_osd.c:64]: Codacy found an issue: [Does not check for buffer overflows when copying to destination (CWE-120).  Make sure destination can always hold the source data. ](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=6303053)

File: src/main/cms/cms_menu_osd.c [hunk 4]
```diff
@@ -44,25 +44,36 @@
 
 #ifdef USE_EXTENDED_CMS_MENUS
 static uint16_t osdConfig_item_pos[OSD_ITEM_COUNT];
+static uint8_t osdConfig_one;
 
 static long menuOsdActiveElemsOnEnter(void)
 {
+    memcpy(&osdConfig_one, &osdConfig()->lq_format, sizeof(osdConfig_t));
```
Comment (nerdCopter) [inline src/main/cms/cms_menu_osd.c:51]: Codacy found an issue: [Does not check for buffer overflows when copying to destination (CWE-120).  Make sure destination can always hold the source data. ](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=6303053)

**Comment by nerdCopter** [comment]
* conflicts, please branch from master

**Review by nerdCopter — APPROVED** [review]
* Kaio reported good

**Pull request merged**